tests/compile-fail/concurrency/libc_pthread_create_main_terminate.rs

```diff
@@ -0,0 +1,24 @@
+// ignore-windows: Concurrency on Windows is not supported yet.
+
+// Check that we terminate the program when the main thread terminates.
+
+//~^^^^ ERROR: unsupported operation: the main thread terminated without waiting for other threads
```

Comment:
What is this?  There's no code up there this is pointing to?
Looks to me like it should use `error-pattern`.